Source/ViewControllers/BrickViewController.swift

```diff
@@ -13,7 +13,7 @@ import UIKit
  Conform to this delegate on your preview view controller in order to enable UIKit Pop.
```

Comment:
Just this protocol shouldn't be restricted to iOS. If it is iOS and tvOS cannot share the same class